src/signals/incident-management/containers/IncidentDetail/components/MetaList/index.test.js

```diff
@@ -70,13 +70,13 @@ describe('<MetaList />', () => {
 
     it('should call onPatchIncident', async () => {
       const { getAllByTestId } = render(withAppContext(<MetaList {...props} />));
-      const editButtons = getAllByTestId('editButton');
+      const editButtons = getAllByTestId('editPriorityButton');
```

Comment:
Same here. Cost some time to figure out where the test-id comes from. We should find a way to easy figure out the id element which is selected